central/main.go

```diff
@@ -486,6 +487,10 @@ func startGRPCServer() {
 
 	basicAuthProvider := userpass.RegisterAuthProviderOrPanic(authProviderRegisteringCtx, basicAuthMgr, registry)
 
+	if features.DeclarativeConfiguration.Enabled() {
+		declarativeconfig.Singleton().WatchDeclarativeConfigDir()
```

Comment:
I don't think I completely get why we need to split the `Singleton` call from `WatchDeclarativeConfigDir()`. Perhaps I miss the bigger picture and there will be more functions added to `declarativeconfig.Manager` interface?

Reply:
I'll be honest, I cannot answer yet in what shape / way it will evolve, I suppose we will see this soon.

While I can move the watch start to the `Singleton()`, without an additional function it would look a bit weird `_ = declarativeconfig.Singleton()`, so let's stick with it, see how the interface evolves, and if we don't see the need for anything additional later on revisit this?

Reply:
Sounds good.